enterprise/frontend/src/metabase-enterprise/content_translation/components/ContentTranslationConfiguration.tsx

```diff
@@ -0,0 +1,158 @@
+import {
+  type ChangeEvent,
+  type ReactNode,
+  useCallback,
+  useRef,
+  useState,
+} from "react";
+import { t } from "ttag";
+
+import { UploadInput } from "metabase/components/upload";
+import ExternalLink from "metabase/core/components/ExternalLink";
+import Markdown from "metabase/core/components/Markdown";
+import {
+  Form,
+  FormProvider,
+  FormSubmitButton,
+  useFormContext,
+} from "metabase/forms";
+import { useSelector } from "metabase/lib/redux";
+import { getDocsUrl } from "metabase/selectors/settings";
+import { Group, Icon, Loader, Stack, Text } from "metabase/ui";
+import { useUploadContentTranslationDictionaryMutation } from "metabase-enterprise/api";
+
+export const ContentTranslationConfiguration = () => {
+  const availableLocalesDocsUrl = useSelector((state) =>
+    // eslint-disable-next-line no-unconditional-metabase-links-render -- This is used in admin settings
+    getDocsUrl(state, {
+      page: "configuring-metabase/localization",
+      anchor: "supported-languages",
+    }),
+  );
+
```

Comment:
Looks like your list endpoint is implemented, but is not actually used.

At the very least, we need to show the user whether there are existing localizations that will be overwritten
- minimum: whether or not there are localizations that will be overwritten
- better: a count of the localizations that will be overwritten
- even better: a count of the localizations by language that will be overwritten
- even more better: who/when uploaded these localizations (probably not possible with current api)
- even more beterer: a modal that just shows a table of the existing localizations


![Screenshot 2025-05-02 at 8 42 54 AM](https://github.com/user-attachments/assets/055fc3e1-e356-4d16-bc9a-e44f5540018f)

Reply:
Just making a note here that, as discussed on Slack, we'll do this in a later PR